changelog/rdmdMakefiles.dd

```diff
@@ -0,0 +1,12 @@
+rdmd can now be used as a shell in makefiles
+
+With gnu make(3.82 or higher), rdmd can now be used in makefiles.
+This is accomplished by setting the SHELL and .SHELLFLAGS to /usr/bin/rdmd and --eval respectively.
+---
+.ONESHELL:
+SHELL = /usr/bin/rdmd
+.SHELLFLAGS = --eval
+hello.txt:
+	import std.file;
+	write("$@","hello world\n");
```

Comment:
FYI: @CyberShadow we should use tabs here, s.t. users can easily copy/paste this file.